### PR TITLE
Robert/zerox docker

### PR DESCRIPTION
**Pull request by robert-e-davidson3**

### CODE REVIEW

File: packages/augur-artifacts/src/index.ts [hunk 1]
```diff
@@ -58,13 +58,23 @@ export interface ContractAddresses {
...
     GnosisSafe?: string;
     ProxyFactory?: string;
-    ZeroXTrade?: string;
-    ZeroXExchange?: string;
     BuyParticipationTokens?: string;
     RedeemStake?: string;
     CashFaucet?: string;
     GnosisSafeRegistry?: string;
     HotLoading?: string;
+    ZeroXTrade?: string; // TODO sup with this?
```
Comment (nuevoalex) [inline packages/augur-artifacts/src/index.ts]: Whats the comment here asking?

**Review by nuevoalex — APPROVED** [review]
LGTM. Confused on one comment

File: packages/augur-artifacts/src/index.ts (hunk 1 above)
Comment (robert-e-davidson3) [reply packages/augur-artifacts/src/index.ts]: heheh I need to do a TODO sweep... At the time I wasn't sure if ZeroXTrade was still needed once we had the official 0x contracts.

Comment (robert-e-davidson3) [reply packages/augur-artifacts/src/index.ts]: That looks like only TODO.

**Pull request merged**